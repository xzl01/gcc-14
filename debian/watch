version=4
ftp://gcc.gnu.org/pub/gcc/releases/gcc-(14\.[\d\.]*)/ \
        gcc-([\d\.]+)\.tar\.xz debian uupdate
