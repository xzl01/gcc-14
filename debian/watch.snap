version=4
ftp://gcc.gnu.org/pub/gcc/snapshots/(15-[\d]+)/ \
        gcc-15-([\d\.]+)\.tar\.xz debian uupdate
